Simple Notepad/README.md

```diff
@@ -0,0 +1,14 @@
+# Package/Script Name
+
+Using Tkinter GUI of simplenotepad is made.
+
+## Setup instructions
+
+Just run script and you are good to go.
+
+## Output
+![simplenotepad](https://cdn1.bbcode0.com/uploads/2021/1/28/8c8c0aad51d75a5b6cb9db6a2a775118-full.png)
```

Comment:
Add link of the image that you have added.

Reply:
Okay , I have fixed it now please check it.